src/css/index.scss

```diff
@@ -232,38 +208,19 @@ h3 {
         font-size: 1.1rem;
     }
     .fullContainer {
-        position:relative;
         display: grid;
         min-width: 400px;
```

Comment:
the min-width was my doing and caused problems. Could you remove it for me or adjust it? I'd appreciate it.